problems.md

```diff
@@ -55,6 +55,7 @@ phone-number |  option, format, unwrap_or, iters, match
 wordy | Result, string parsing, operators (optional)
 tournament |  enum, sorting, hashmap, structs
 custom-set |  generic over type, vector, equality, struct
+atbash-cipher | str vs string, primitive types, iterators, chars, ascii
```

Comment:
don't forget to move this too, we'd like it to be in the same place relative to other problems as it is in `config.json`